modules/src/core/ics02_client/context.rs

```diff
@@ -117,21 +128,20 @@ pub trait ClientKeeper {
                                     height,
                                     cs_state,
                                 )?;
+                                self.store_update_time(
+                                    res.client_id.clone(),
+                                    height,
+                                    res.processed_time,
+                                )?;
+                                self.store_update_height(
+                                    res.client_id.clone(),
+                                    height,
+                                    res.processed_height,
+                                )?;
```

Comment:
don't we only want to store the most recent height?

Reply:
We need to be able to retrieve the time a client was updated for every height to adequately verify if the delay period was observed